DesignAutomationHandler.cs

```diff
@@ -19,7 +19,16 @@ public Result Execute(ExternalCommandData commandData, ref string message, Eleme
             string[] files = Directory.GetFiles(app.AllUsersAddinsLocation, "*.addin");
             foreach (string file in files)
             {
-               XElement addin = XElement.Load(file);
+               XElement addin;
+               try
+               {
+                  addin = XElement.Load(file);
+               }
+               catch
+               {
+                  // if the XML is malformed or not parse-able, just continue
+                  continue;
```

Comment:
Does it ever happen that `*.addin` is malformed? 
Instead of ignoring the error in silence, we'd better at least pop out an error message by "MessageBox.Show"(for example).

Reply:
It seems like it does, see issue filed last year here: https://github.com/Autodesk-Forge/design.automation-csharp-revit.local.debug.tool/issues/8
We also see some issues on stackoverflow: https://stackoverflow.com/questions/76132803/forge-design-automation-debug-revit-plugin-locally-tutorial-not-working

(however, we can certainly throw an message)